client/components/community-translator/style.scss

```diff
@@ -0,0 +1,81 @@
+.community-translator__element {
+	display: inline-block;
+	cursor: pointer;
+	z-index: 100;
```

Comment:
When it comes to z-index, there would be less potential future z-conflicts if we reuse the [z-index](https://github.com/Automattic/wp-calypso/blob/master/assets/stylesheets/shared/functions/_z-index.scss#L297) scss function.

Reply:
Good call on using the function 👍